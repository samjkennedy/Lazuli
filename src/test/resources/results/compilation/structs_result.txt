1.0
1.0
1.414213562373095
1.0
2.0
3.0
3.7416573867739413
