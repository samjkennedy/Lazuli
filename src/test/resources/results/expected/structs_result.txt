1.0
1.0
1.414213562373095
1.0
2.0
3.0
3.7416573867739413
Hello
Goodbye
7
8
false
true
1.141
3.1415
